docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: '3'
+services:
+  jenkins:
+    image: jenkins/jenkins:lts
+    ports:
+      - 8080:8080 #expose port 8080 to host
+    volumes:
```

Comment:
Can you also figure out a way to mount roles as well? I am sure that we can create folder mounts inside docker volumes mounts.

So, something like 

```
volumes:
     JENKINS_DATA:/var/jenkins_home
     jobs/RCW:/var/jenkins_home/jobs/RCW
```

will work. You would have to figure out a way to fix the permission problem.

Reply:
Sure, go ahead and see if this works.